src/app/pages/terms-and-conditions/terms-and-conditions.page.html

```diff
@@ -0,0 +1,20 @@
+<ion-content class="ion-padding">
+  <app-generic-header [label1]="'TERMS_AND'" [label2]="'CONDITIONS'"></app-generic-header>
+  <div *ngFor="let item of items">
+    <app-terms-and-condition-card expandHeight="150px" (click)="expandItem(item)" [item]="item"
```

Comment:
the component name should not be specific to the feature. Give a genric name so that we can reuse it anywhere

Reply:
ok